cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -485,14 +485,16 @@ if("${DOUBLE_CONVERSION_HOME}" STREQUAL "")
   set(DOUBLE_CONVERSION_VENDORED 1)
 else()
   find_package(double-conversion REQUIRED)
+  set(DOUBLE_CONVERSION_TARGET double-conversion::double-conversion)
```

Comment:
I'm not fond of the indirection here. Is there possibly a way to alias `double-conversion::double-conversion` to `double-conversion`?

Reply:
There is
```cmake
add_library(double-conversion ALIAS double-conversion::double-conversion)
```
which doesn't work:
```
CMake Error at cmake_modules/ThirdpartyToolchain.cmake:472 (add_library):
  add_library cannot create ALIAS target "double-conversion" because target
  "double-conversion::double-conversion" is imported but not globally
  visible.
```
I implemented another solution (see new commit below).